.nf-core.yml

```diff
@@ -1,5 +1,6 @@
 files_unchanged:
   - lib/NfcoreSchema.groovy
+  - assets/multiqc_config.yaml
 actions_awsfulltest: False
 params_used: False
 files_exist:
```

Comment:
```suggestion
lint:
  files_unchanged:
    - assets/multiqc_config.yaml
```

Reply:
Ah sorry, that was my fault, the structure of this file changed and there needs to be added `lint:`